### PR TITLE
report written keys to pd

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
@siddontang @nolouch PTAL

### CODE REVIEW

File: src/pd/async/client.rs [hunk 1]
```diff
@@ -191,15 +191,16 @@ impl PdClient for RpcClient {
                         leader: metapb::Peer,
                         down_peers: Vec<pdpb::PeerStats>,
                         pending_peers: Vec<metapb::Peer>,
-                        written_bytes: u64)
+                        region_stat: RegionStat)
```
Comment (siddontang) [inline src/pd/async/client.rs]: I prefer to put all fields like down_peers, pending_peers to the RegionStat like #1765 does.

Comment (ngaut) [inline src/pd/async/client.rs]: Me too.

**Comment by zhangjinpeng1987** [comment]
PTAL

**Comment by nolouch** [comment]
LGTM

**Pull request merged**